synapse/cortex.py

```diff
@@ -49,6 +49,7 @@
 import synapse.lib.stormlib.auth as s_stormlib_auth # NOQA
 import synapse.lib.stormlib.cell as s_stormlib_cell # NOQA
 import synapse.lib.stormlib.json as s_stormlib_json  # NOQA
+import synapse.lib.stormlib.smtp as s_stormlib_stix  # NOQA
```

Comment:
Did you mean to have this as `s_stormlib_smtp` instead?

Reply:
yep!  thanks!